src/dms/core/EdFi.DataManagementService.Core.External/Model/AuthorizationFilter.cs

```diff
@@ -9,13 +9,62 @@ namespace EdFi.DataManagementService.Core.External.Model;
 /// Represents an authorization filter with a specified
 /// JSON element name for extracting data, and the expected value
 /// </summary>
-public record AuthorizationFilter(
+public abstract record AuthorizationFilter(
     /// <summary>
-    /// The JSON element name used to extract the relevant data
+    /// The column name in Document and the element name in the search engine index
     /// </summary>
-    string FilterPath,
+    string FilteringMetadataElementName,
     /// <summary>
     /// The expected value used
     /// </summary>
     string Value
-);
+)
+{
+    /// <summary>
+    /// Education Organization based authorization filter
+    /// <param name="FilteringMetadataElementName">The column name in Document and the element name in the search engine index</param>
+    /// <param name="EducationOrganizationId"></param>
+    /// </summary>
+    public record EducationOrganization(string FilteringMetadataElementName, string EducationOrganizationId)
```

Comment:
Do we even need the `FilteringMetadataElementName` parameter on any of these records? Would it ever be correct for an `EducationOrganization` to have one not named "EducationOrganization"?